conf/application.conf

```diff
@@ -15,6 +15,16 @@
 # HOCON will fall back to substituting environment variable:
 #mykey = ${JAVA_HOME}
 
+## Digitalisiertedrucke
+
+index {
+	host = "localhost"
+	name = "digitalisiertedrucke"
+	location = "."
+	http_port = "6011"
```

Comment:
The notation with dots is equivalent to the nesting, i.e. the nested properties here can be accesses with `index.host` etc. In organisations, it's actually very deeply nested: `index.es.port.http` is actually 4 levels deep. That would not look very clean if written nested. On the other hand, if it's only one level like here, I think its very readable and provides nice grouping. So I'd say it depends on the concrete use case which notation makes more sense.